### DISCUSSION TITLE
Preferences

### DISCUSSION THREAD

**Discussion by smk78** [SasView/sasview#2560] [Ideas]
Before we go too far down the road of the Preferences panel (which I think is a nice feature), perhaps we could have a discussion about what is currently in it, and what else might be included?

For example, perhaps the user could select what the default behaviour of the Change Data Name option in the Data Explorer context menu is? If I recall, we changed the data label _from_ the file name _to_ the data name because someone thought that was more logical. But yesterday I was shown a use case where that became unhelpful...

**Comment by lucas-wilkins** [comment]
I have my eye on the data panel in general, one of the big things I want to address at the codecamp with respect to look and feel.

If you look at how IDEs deal with this problem, there's generally two (or more) views, one for files, one for data objects.
